GOVERNANCE.rst

```diff
@@ -335,16 +346,51 @@ sktime uses a “consensus seeking” process for making decisions. The
 community tries to find a resolution that has no open objections among
 core developers.
 
--  To accept proposed changes, we require approval by one core developer
-   (lazy consensus) and no rejection by a core developer (veto right).
+-  For a proposed changes to be approved via lazy consensus, one of the following
+   two conditions must be satisfied:
+    - approval by at least one core developer and no rejection by a core developer (veto right).
+      The approval required for this condition must be by a core developer different from a proposer.
+    - approval in the regular core developer meeting as part of the agenda.
+      This can happen only if no approvals or rejections are expresesd on a PR, see below.
+-  For a proposed change to be rejected via lazy consensus, one of the following two conditions
+   must be satisfied:
+    - rejection by at least one core developer, and no acceptance by a core developer.
+    - rejection in the regular core developer meeting as part of the agenda.
+      This can happen only if no approvals or rejections are expresesd on a PR, see below.
```

Comment:
```suggestion
```

Reply:
no? I thought we agreed that if no approvals/rejections appear, it is tabled at the top of agenda at the core dev meeting.

Reply:
simplified